test/job-test.js

```diff
@@ -161,6 +161,19 @@ describe('Job', (it) => {
       t.is(await hget(queue.toKey('jobs'), job.id), null);
     });
 
+    it('should warn about job being removed', async (t) => {
```

Comment:
How does the test assert the warning?

Reply:
I meant "notified", like an event is fired when the job gets removed. 

So, it's not clear to me: I though you didn't want to accept the PR, from your previous comment